cluster_tools/cluster_tools/schedulers/cluster_executor.py

```diff
@@ -128,6 +128,7 @@ def handle_kill(self, existing_sigint_handler, signum, frame):
         if (
             existing_sigint_handler  # pylint: disable=comparison-with-callable
             != signal.default_int_handler
+            and callable(existing_sigint_handler)  # Could also be signal.SIG_IGN
         ):
             existing_sigint_handler(signum, frame)
```

Comment:
So, `signal.SIG_IGN` is the 'Handlers' object which is mentioned in the stack trace?

Reply:
Yep:

```
>>> type(signal.SIG_IGN)
<enum 'Handlers'>
```